dist/api_data.py

```diff
@@ -1430,6 +1430,10 @@ def __ge__(self, other):
 ]),
 
 'WT_SESSION.begin_transaction' : Method([
+    Config('read_before_oldest', 'false', r'''
+        allows the caller to specify a read timestamp less than the oldest timestamp but newer
+        than or equal to the pinned timestamp, cannot be set to true while also rounding up the read
```

Comment:
"... pinned timestamp, cannot..." Those should be separate sentences. "pinned timestamp. Cannot be set..."